SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf

```diff
@@ -127,7 +127,7 @@
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-sec-dbt-ab-mbs;AgencyMBSDeal">
-		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-mbs;MortgageBackedSecurityOffering"/>
+		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-dbti;DebtOffering"/>
 		<rdfs:label xml:lang="en">agency m b s deal</rdfs:label>
```

Comment:
We should use MBS in the label

Reply:
@rivettp I think this may go away altogether and there are many cases in the MBS ontology of the misuse of abbreviations that we have yet to address.